internals/cli/export_test.go

```diff
@@ -20,9 +20,7 @@ import (
 	"github.com/canonical/pebble/client"
 )
 
-var RunMain = Run
-
-var ClientConfig = &clientConfig
+var ClientConfig *client.Config = &client.Config{}
```

Comment:
Per above, I'm not sure why we'd need to export this, instead of using `&client.Config{}` directly.

Reply:
Because at one point some test expect this to be initialized after a call to `cli.Run()` from within the tests, and those fail without a valid client config initialized with an open Pebble socket.